samples/Erlang/sample.app.src

```diff
@@ -0,0 +1,8 @@
+{application, sample,
```

Comment:
Unless I'm missing something, it should highlight correctly.  It's the same syntax.

If a mode-line `-*- mode: erlang; -*-` comment is added to the top of the file, [it works correctly](https://github.com/erlang/rebar3/blob/master/src/rebar.app.src).